src/Objects/MonitorManager.vala

```diff
@@ -66,12 +66,22 @@ public class Display.MonitorManager : GLib.Object {
         virtual_monitors = new Gee.LinkedList<Display.VirtualMonitor> ();
         try {
             iface = Bus.get_proxy_sync (BusType.SESSION, "org.gnome.Mutter.DisplayConfig", "/org/gnome/Mutter/DisplayConfig");
-            iface.monitors_changed.connect (get_monitor_config);
+            iface.monitors_changed.connect (on_monitors_changed);
```

Comment:
Better name `on_iface_monitors_changed` to distinguish from local signal with a similar name.

Reply:
Done.